cdap-ui/app/cdap/components/Experiments/CreateView/index.js

```diff
@@ -115,6 +114,15 @@ export default class ExperimentCreateView extends Component {
     fetchAlgorithmsList();
   }
   componentWillUnmount() {
+    let {experiments_create} = createExperimentStore.getState();
+    let {workspaceId} = experiments_create;
+    if (workspaceId) {
+      // Every workspace created in experiments create view is temp. So don't worry about deleting it.
+      MyDataPrepApi.delete({
+        namespace: getCurrentNamespace(),
+        workspaceId
+      }).subscribe();
```

Comment:
What's the use of the empty `subscribe`?

Reply:
`Rx` won't make a query unless it has a subscription. That is the one of the key difference between a promise and a Observable. So you have to have a dummy subscription even if you don't need the response for rx to make the call for us.